popup.js

```diff
@@ -199,6 +216,39 @@ function onOptionChange(elem) {
   changeFont();
 }
 
+/**
+ * Helper method to remove duplicate strings from an array (case-insensitive).
+ * @param {Array} stringArray
+ */
+ function removeDuplicates(stringArray) {
+  return [...new Set(
+    stringArray.map(str => str.toLowerCase())
+  )];
+}
+
+/**
+ * Helper method that returns true if a given font family matches any search term in a list (or the list is empty).
+ * @param {Array} searchTermArray
+ * @param {String} fontFamily
+ */
+function doesSearchTermMatch(searchTermArray, fontFamily) {
+  return searchTermArray.length === 0 || searchTermArray.some(searchTerm => fontFamily.includes(searchTerm));
```

Comment:
ES6 has pretty good support nowadays. As far as I'm concerned, if a browser supports Web Extensions it's going to support ES6 features. CanIUse says Chrome has had full support since mid-2016 and other browsers are similar: https://caniuse.com/es6